docs/operator-guide/troubleshooting/troubleshoot-container-registries.md

```diff
@@ -0,0 +1,25 @@
+# Container Registry Troubleshooting
+
+## Problem
+
+Reset container registry is not feasible due to the `Change registry` button not working.
+
+## Cause
+
+The External Secrets Operator (ESO) doesn't allow you to reconfigure registry.
+
+## Solution
+
+Remove the `kaniko-docker-config` and `regcred` resources from both ESO and secrets.
+
+1. Check the secrets in the External Secrets Operator:
+
+2. Delete secrets using the commands below:
+
+3. Delete `kaniko-docker-config` and `regcred` secrets:
+
+4. Refresh the KubeRocketCI portal page and set your container registry again.
+
```

Comment:
1. Check the `kaniko-docker-config` and `regcred` ExternalSecret custom resources (CRs) in the namespace:
```bash
kubectl placeholder
```
2. Delete the `kaniko-docker-config` and `regcred` ExternalSecret CRs using the commands provided below:
```bash
kubectl placeholder
```
3. Delete the `kaniko-docker-config` and `regcred` Kubernetes secrets if they have not been automatically deleted by the External Secrets Operator:
```bash
kubectl placeholder
```
4. Refresh the KubeRocketCI portal page and reconfigure your container registry.

Reply:
please align with this version ^